.github/workflows/CI.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - master
       - release-*
+    tags: '*'
```

Comment:
@fredrikekre I think this is what we need to get the tags working.

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

Reply:
Correct https://github.com/jump-dev/JuMP.jl/blob/9c4dbfd829487af5dac8c45f89e4de7c7496e2a8/.github/workflows/documentation.yml#L5